examples/GLLinePlotItem.py

```diff
@@ -30,14 +30,14 @@
 w.addItem(gz)
 
 def fn(x, y):
```

Comment:
This function is not used.

Reply:
Realized this just after I committed the code.